src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
+  var ageDifMs = Date.now() - birthday;
+  var ageDate = new Date(ageDifMs); // miliseconds from epoch
+  return Math.abs(ageDate.getUTCFullYear() - 1970);
+};
+interface Patient {
```

Comment:
Could you put this interface at the top of the file? Or even in a dedicated file if you think it's worth it

Reply:
Moved in a dedicated file